crypto/source_address_token.go

```diff
@@ -58,7 +58,11 @@ const stkKeySize = 16
 const stkNonceSize = 16
 
 // NewStkSource creates a source for source address tokens
-func NewStkSource(secret []byte) (StkSource, error) {
+func NewStkSource() (StkSource, error) {
```

Comment:
The STK doesn't have to do anything with the server config. In fact, even when the server config changes, we still want to be able to read STKs issues before that change.